eis/vocabs/readme.md

```diff
@@ -0,0 +1,19 @@
+Repository for redirecting to videogame extension vocabularies
+===================
+
+Repository created to store the redirections of the the vocabulary for AutomationML
+
+At the moment we have three games extensions:
+
+/eis/vocabs/aml: 
+
+Html: https://github.com/i40-Tools/vocabularies/tree/master/AutomationML/amlDocs/
```

Comment:
Yes, what I want to achieve is that when I type this https://w3id.org/eis/vocabs/aml# it load the documentation page of the vocabulary. I tried to follow the steps to do so by no success so far.